components/toast/index.css

```diff
@@ -174,24 +146,8 @@
 	align-items: center;
 	flex: 1 1 auto;
 	align-self: center;
-
-	padding-block-start: var(--mod-toast-spacing-block-start, var(--spectrum-toast-spacing-block-start));
-	padding-block-end: var(--mod-toast-spacing-block-end, var(--spectrum-toast-spacing-block-end));
-
-	.spectrum-Button {
-		margin-inline-start: auto;
-		margin-inline-end: var(--mod-toast-spacing-text-and-action-button-to-divider, var(--spectrum-toast-spacing-text-and-action-button-to-divider));
-
-		&:dir(rtl) {
-			margin-inline-end: auto;
-			margin-inline-end: var(--mod-toast-spacing-text-and-action-button-to-divider, var(--spectrum-toast-spacing-text-and-action-button-to-divider));
-		}
-	}
-
-	& + .spectrum-Toast-buttons {
-		padding-inline-start: var(--mod-toast-spacing-close-button, var(--spectrum-toast-spacing-close-button));
-
-		border-inline-start-width: 1px;
-		border-inline-start-style: solid;
-	}
+	column-gap: var(--mod-toast-spacing-text-to-button-horizontal, var(--spectrum-toast-spacing-text-to-button-horizontal));
+	row-gap: var(--mod-toast-spacing-text-to-button-vertical, var(--spectrum-toast-spacing-text-to-button-vertical));
+	padding-block-start: var(--mod-toast-spacing-top-edge-to-text, var(--spectrum-toast-spacing-top-edge-to-text));
+	padding-block-end: var(--mod-toast-spacing-bottom-edge-to-text, var(--spectrum-toast-spacing-bottom-edge-to-text));
```

Comment:
I wonder if we can incorporate the alert banner's `max` function in here (or somewhere...) to handle the oddities in the padding and wrapping and all that. Maybe we need an additional `p` in the template around the toast text? I'll play around with some of it and see if I can come up with something.